### ISSUE TITLE
 if the learning rate adjustment steps need to change

### ISSUE THREAD

**Issue by zhaoxiaochuang001** [finger-monkey/Data-Augmentation#6]
Hi,Thanks for your code!
I want to know when using your code for training, besides adding training epochs, if the learning rate adjustment steps need to change?In Reid's code, the learning rate adjustment steps is [40,70]. I would like to know what adjustment steps you use in training.

**Comment by finger-monkey** [comment]
hi, besides adding training epochs, I  have not adjusted any other parameters, but you can try it and it might work better.

**Comment by zhaoxiaochuang001** [comment]
Thanks for your reply!
